athena-dynamodb/athena-dynamodb.yaml

```diff
@@ -29,8 +29,8 @@ Parameters:
     Default: 900
     Type: Number
   LambdaMemory:
-    Description: 'Lambda memory in MB (min 128 - 3008 max).'
-    Default: 3008
+    Description: 'Lambda memory in MB (min 128 - 10240 max).'
+    Default: 10240
```

Comment:
I don't think we should change the default, as that will incur more cost

Reply:
Yes, agree.

Will take back the PR.